dsaps/cli.py

```diff
@@ -151,18 +141,40 @@ def additems(
     elif "collection_uuid" in ctx.obj:
         collection_uuid = ctx.obj["collection_uuid"]
     else:
-        collection_uuid = client.get_uuid_from_handle(collection_handle)
-    with open(metadata_csv, "r") as csvfile, open(field_map, "r") as jsonfile:
+        collection_uuid = dspace_client.get_uuid_from_handle(collection_handle)
+
+    if metadata_csv is None:
+        raise click.UsageError(
+            "Option '--metadata-csv' must be used or " "run 'reconcile' before 'additems'"
+        )
+
+    bitstream_file_paths = helpers.get_bitstreams_from_csv(metadata_csv)
```

Comment:
I've considered and reconsidered but I think it would be better to create and attach the `Bitstreams` in `metadata_from_csv_row`. We should break that method up a bit (probably should have earlier) but that way we're only accessing the `metadata_csv` once to handle metadata and bitstreams. Happy to talk it out when we meet later

Reply:
@ehanson8 That is a very good point, and I think is the reason why I had added `bitstreams` as an attribute to the `Item` class at some point (but did not do anything with it). Let me know what you think of the additional updates proposed in the latest commit https://github.com/MITLibraries/dspace-api-python-scripts/pull/39/commits/8f61a2ad7edc5d017b9c8e9f364acfddca601e81. Happy to discuss during our sync this afternoon!